src/quacc/recipes/newtonnet/ts.py

```diff
@@ -266,6 +282,269 @@ def quasi_irc_job(
     return relax_summary
 
 
+@job
+@requires(
+    has_newtonnet, "NewtonNet must be installed. Refer to the quacc documentation."
+)
+@requires(
+    has_geodesic_interpolate,
+    "geodesic-interpolate must be installed. Refer to the quacc documentation.",
+)
+def neb_job(
+    reactant_atoms: Atoms,
+    product_atoms: Atoms,
+    relax_job_kwargs: dict[str, Any] | None = None,
+    calc_kwargs: dict[str, Any] | None = None,
+    geodesic_interpolate_kwargs: dict[str, Any] | None = None,
+    neb_kwargs: dict[str, Any] | None = None,
+) -> NebSchema:
+    """
+    Perform a nudged elastic band (NEB) calculation to find the minimum energy path (MEP) between the given reactant and product structures.
+
+    Parameters
+    ----------
+    reactant_atoms
+        The Atoms object representing the reactant structure.
+    product_atoms
+        The Atoms object representing the product structure.
+    relax_job_kwargs
+        Keyword arguments to use relax_job.
+    calc_kwargs
+        Custom kwargs for the NewtonNet calculator.
+    geodesic_interpolate_kwargs
+        Keyword arguments for the geodesic function.
+    neb_kwargs
+        Keyword arguments for the NEB calculation.
```

Comment:
This comment applies to both functions:

As done in other recipes throughout quacc, please clearly cite which functions or classes you are referring to. For instance, "Keyword arguments to use [quacc.recipes.newtonnet.core.relax_job][]` is much clearer than just "relax_job." It will also automatically hyperlink in the built docs. Do this anytime you have a "kwargs" as an argument.

Reply:
I changed a few of them. Some of them like the geodesic interpolate kwargs did not work and gave errors in docs so I got rid of them.